packages/fetch-impl/src/dispatcher.ts

```diff
@@ -0,0 +1,16 @@
+import { Agent, Dispatcher } from 'undici';
+
+let cachedDispatcher: Agent;
+export function getDefaultDispatcher(): Dispatcher {
+    if (!cachedDispatcher) {
+        cachedDispatcher = new Agent({
+            // FIXME: https://github.com/nodejs/undici/issues/2808
+            // allowH2: true,
+            connections: 16,
+            // One second fewer than the Solana RPC's keepalive timeout.
+            // Read more: https://github.com/solana-labs/solana/issues/27859#issuecomment-1340097889
+            keepAliveTimeout: 19000,
+        });
```

Comment:
I made the decision to deliver a default dispatcher to everyone. This is what the current web3.js does. I _really_ hate this, because everyone's workload is different and requires tuning, but it's also true that having _no_ dispatching strategy has pretty awful performance.